### PR TITLE
update script for runninglinuxkernel_4.0

### PR DESCRIPTION
**Pull request by miraclemore**
1. Add script for qemu boot kernel command
2. Enable virt-9p to support folder shared
3. add /mnt and /lib/modules/4.0.0+ for rootfs
4. Currently only support x86 platform
5. fix some build warnning

### CODE REVIEW

**Comment by miraclemore** [comment]
Dear figo,
please help to review that.
if any problems, please let me know.
thx!

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+LROOT=$PWD
+ROOTFS_X86=_install_x86
+ROOTFS_ARM=_install_arm
```
Comment (figozhang) [inline run.sh]: should be _install_arm32

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                        -dtb arch/arm/boot/dts/vexpress-v2p-ca9.dtb -nographic \
+                        -append "rdinit=/linuxrc console=ttyAMA0 loglevel=8" \
+                        $DBG ;;
+    arm64)
+	if [ ! -c $LROOT/$ROOTFS_ARM64/$CONSOLE_DEV_NODE ]; then
+		echo "please create console device node first, and recompile kernel"
+		exit 1
+	fi
+        qemu-system-aarch64 -machine virt -cpu cortex-a57 -machine type=virt \
+                            -m 2048 –smp 2 -kernel arch/arm64/boot/Image \
```
Comment (figozhang) [inline run.sh]: benshushu@ubuntu:~/work/runninglinuxkernel_4.0$ sh run.sh arm64
qemu-system-aarch64: -m 2048: Could not open '–smp': No such file or directory
benshushu@ubuntu:~/work/runninglinuxkernel_4.0$ vim run.sh

**Comment by figozhang** [comment]
thanks a lot for your patch. pls check arm and arm64 can work before you send the patch.

File: run.sh (hunk 1 above)
Comment (miraclemore) [reply run.sh]: As I comment above on the patch.
Currently only support x86 platform. I will support that later

**Pull request closed without merging**